tests/test_web_log.py

```diff
@@ -251,3 +252,20 @@ def log(self, request, response, time):
     resp = await client.get("/")
     assert 200 == resp.status
     assert msg == "contextvars: uuid"
+
+
+def test_logger_does_nothing_when_disabled(caplog: pytest.LogCaptureFixture) -> None:
```

Comment:
Doesn't this just end up testing the logging module?
i.e. This test surely passes without the above patch, because if the logging level is set to WARNING then the logger won't log anything if you call it with a log level of INFO.

Reply:
The test is only validating that we don't break existing behavior. It doesn't validate that the logger call isn't run. It seemed messy to patch the logger to validate that.

Reply:
Right, I guess it verifies that the log is still happening in the positive case. The negative case is a bit redundant though as the code couldn't possibly affect it.